src/tests/ftest/ior/crash.py

```diff
@@ -62,6 +62,21 @@ def test_ior_crash(self):
         if not check_system_query_status(scan_info):
             self.fail("One or more engines crashed")
 
+        # Verify container handle opened by ior is closed (by daos_agent after ior crash).
+        # Expect one handle (associated with this process check_container_info below).
+        # Note: it is possible query may run before handles are closed, and fail this test.
+        # Though, let us assume the dmg system query command above will take up sufficient time.
+        checks = {
+            "ci_nhandles": 1}
+        attempts = 0
+        while attempts < 5:
+            attempts += 1
+            chkres = self.container.check_container_info(**checks)
+            if chkres is True:
+                break
+            time.sleep(2)
```

Comment:
FYI - suggestion - python allows this syntax:
```suggestion
        for _ in range(5):
            chkres = self.container.check_container_info(**checks)
            if chkres is True:
                break
            time.sleep(2)
```

Reply:
done in a new helper method, thanks